src/mon/OSDMonitor.cc

```diff
@@ -831,8 +831,7 @@ void OSDMonitor::prime_pg_temp(
   const OSDMap& next,
   pg_t pgid)
 {
-  // TODO: remove this creating_pgs direct access?
-  if (creating_pgs.pgs.count(pgid)) {
+  if (get_creating_pgs_size(pgid)) {
```

Comment:
there are more places where we are referencing `creating_pgs.pg` in this class. i don't think removing this direct access helps in the very case helps. neither do i think we need to do so. as `creating_pgs_t` is pretty much a POD data class.